src/server/render/tree-to-html.js

```diff
@@ -7,14 +7,24 @@ export default async ({
   Component,
   routeOptions = {},
   match,
-  componentData
+  componentData,
+  queryParams
 }) => {
-  const app = <Component {...match} {...componentData} />
-  const loadableState = await getLoadableState(app)
+  const bootstrapData = {
+    ...componentData,
+    requestData: {
```

Comment:
Hey @jonsherrard this is where the object structure changes as now we'd be passing in the component data as well as the new `requestData` object. Where `bootstrapData` will eventually become `window.__TAPESTRY_DATA__.appData`.

Reply:
I wonder if we should nest the `requestData` within a `tapestry`/`tapestryData`/`__tapestry` key? What would be the most obvious key within the WP data that wouldn't clash with any response from WP and is recognisable as Tapestry related

Reply:
I like the idea of a `__tapestry` actually. Maybe modifying the object structure to be:

```
appData: {},
__tapestry: {
  requestData: {}
},
ids: []
```

This would eliminate the need to change the app data object structure and provides a place we can use to store data like this in future?

@thomasdigby @jonsherrard

Reply:
Bear in mind that we only pass in `appData` to the component, so we'll need to include `__tapestry` in that key

Reply:
Actually, let's try and match WP's schema, `_tapestry` should do it.

Reply:
Yep that's a good shout, let's do that 👍